flake.nix

```diff
@@ -7,7 +7,6 @@
 
   outputs = { self }:
     let
-
```

Comment:
as a general rule nixpkgs-fmt doesn't collapse lines, but not 100% sure in this case